src/vcf/report_writer.cpp

```diff
@@ -0,0 +1,120 @@
+//
+// Created by jmmut on 25/05/16.
+//
+
+#include "vcf/report_writer.hpp"
+
+namespace ebi
+{
+  namespace vcf
+  {
+    SqliteReportWriter::SqliteReportWriter(std::string db_name) : db_name{db_name}, current_transaction_size{0},
+                                                      transaction_size{1000000}, sleep_time{500}
+    {
+        int rc = sqlite3_open(db_name.c_str(), &db);
```

Comment:
In order to prevent overwriting any previous output by mistake, and taking into account the conversation we had some days ago about generating a timestamped file for every run, throwing an exception seems like the way to go.